<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JauMoon</class>
 <widget class="QMainWindow" name="JauMoon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jau</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QRadioButton" name="getstrRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1、抽取</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="transRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2、翻译</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="updateRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3、更新</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="querytRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3、查询</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="traveRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5、遍历</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="otherGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>其他选项</string>
    </property>
    <widget class="QCheckBox" name="oldCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>兼容</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="delCheckBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="otherCheckBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>其他</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="instructionGroupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>说明</string>
    </property>
    <widget class="QTextEdit" name="funcTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dirGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>路径</string>
    </property>
    <widget class="QLabel" name="sdirLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>22</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>源路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sdirLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tdirLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tdirLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>目路径：</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sdirToolButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tdirToolButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exePushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelPushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="operateTabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>模式</string>
     </attribute>
     <widget class="QTextEdit" name="modeTextEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="charRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字符串</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="fileRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>单文件</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="batRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>批处理</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>操作</string>
     </attribute>
     <widget class="QRadioButton" name="exportRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="checkvlistRadioButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>检查格式串</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="convertRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>转换</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="checkbracketsRadioButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>检查括号</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="operateTextEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
